soul-admin/src/main/resources/application.yml

```diff
@@ -38,14 +38,15 @@ mybatis:
 
 soul:
   register:
-    registerType: http #http #zookeeper #etcd #consul
-    serverLists : http://localhost:9095 #localhost:2181 #http://localhost:2379
+    registerType: http #http #zookeeper
```

Comment:
why  modify this

Reply:
because it's too long, so I delete some.

I will recover it.